src/Svenkle.NuGetServer.Service/Program.cs

```diff
@@ -25,24 +30,51 @@ public class Service : ServiceBase
         private string _hostFolder;
         private string _host;
 
+        private static Logger logger = LogManager.GetCurrentClassLogger();
+
         private static void Main(string[] args)
         {
             var service = new Service();
+
             if (Environment.UserInteractive)
             {
-                Console.WriteLine($@"[{typeof(Service).Namespace}]");
-                Console.WriteLine(@"Starting...");
-                service.OnStart(args);
-                Console.WriteLine(@"Started!");
-                Console.ReadLine();
-                service.Stop();
+
+                string parameter = string.Concat(args);
+                switch (parameter)
+                {
+                    case "--install":
+                    case "-i":
+                        ManagedInstallerClass.InstallHelper(new string[] { Assembly.GetExecutingAssembly().Location });
+                        break;
+                    case "--uninstall":
+                    case "-u":
+                        ManagedInstallerClass.InstallHelper(new string[] { "/u", Assembly.GetExecutingAssembly().Location });
+                        break;
+                    default:
+                        Console.WriteLine($@"[{typeof(Service).Namespace}]");
```

Comment:
Instead of having both a console.write and logger.write each time use an NLog console and file target - reducing the number of lines required for the same outcome.